playbooks/roles/ocp-install/tasks/checkcert.yaml

```diff
@@ -0,0 +1,32 @@
+---
+# tasks file for ocp4 install cert
+
+- name: check if oc is installed
+  command: type -P oc
+  register: oc_installed
+
+# if cluster is under 24 hrs. old set under24 to 1
+- name: check the cluster timestamp
+  shell: |
+    # get the cluster create time in seconds
+    tmstmp=`date -d$(oc get -o json clusterversion version |  jq -r '.metadata.creationTimestamp') '+%s'`
+    echo $(( $tmstmp  + 86400 > `date +%s` ? 1 : 0 ))
+  register: under24 
```

Comment:
Should not the variable mean `over24hours`?

Reply:
Well can be done. I was checking Not Condition.